package.json

```diff
@@ -67,7 +67,7 @@
     "uglify-js": "^3.4.3"
   },
   "devDependencies": {
-    "@mapbox/batfish": "2.1.0",
+    "@mapbox/batfish": "2.2.0",
```

Comment:
This is not strictly necessary for the node 14 upgrade but I figured I'd sneak it in.